dd-java-agent/instrumentation/jetty-8/jetty-8.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
   testCompile group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '8.0.0.v20110901'
   testCompile group: 'org.eclipse.jetty', name: 'jetty-continuation', version: '8.0.0.v20110901'
 
-  latestDepTestCompile group: 'org.eclipse.jetty', name: 'jetty-server', version: '+'
-  latestDepTestCompile group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '+'
-  latestDepTestCompile group: 'org.eclipse.jetty', name: 'jetty-continuation', version: '+'
+  latestDepTestCompile group: 'org.eclipse.jetty', name: 'jetty-server', version: '9.+'
```

Comment:
Hmm, this is in jetty-8, are we sure that's the right version spec?

Reply:
This instruments jetty 8+, but it looks like the recently released 10 alpha is incompatible and will require updates, possibly to the build system.